app/views/users/show.html.erb

```diff
@@ -8,4 +8,16 @@
       </h1>
     </section>
   </aside>
+  <div class="col-md-8">
+    <% if @user.microposts.any? %>
+      <h3>
+        <%= t ".microposts" %>
+        (<%= @user.microposts.count %>)
```

Comment:
so sánh count, size, lenght, nên dùng cái nào

Reply:
- count: Đếm số lượng record bằng câu lệnh SELECT COUNT(*) FROM... count không lưu trữ trong đối tượng gọi method, mỗi lần gọi count là một lần thực hiện câu lệnh truy vấn SELECT.
- length: Nạp toàn bộ record vào bộ nhớ và thực hiện đếm. Nếu record đã được nạp vào bộ nhớ trước đó thì trực tiếp đếm số lượng.
- size: Kết hợp giữa count và length. Nếu record chưa được load vào bộ nhớ, size thực hiện lệnh SELECT trực tiếp trong CSDL tương tự count, nếu đã load thì đếm tương tự length.

- Tùy vào mục đích để sử dụng count, size hoặc length. 
-- Trong các trường hợp thường gặp nên dùng count, lý do là count tương tác trực tiếp với CSDL, trong khi size và length có thể sai sót (Trong trường hợp đã tạo + lưu record mới nhưng chưa reload bộ nhớ/ Tạo trực tiếp bằng create method mà không thông qua mối quan hệ (association)...)
-- Với vòng lặp thì có thể cân nhắc dùng length hoặc size để không phải truy vấn lại nhiều lần.